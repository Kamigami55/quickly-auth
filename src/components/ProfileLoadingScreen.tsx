import { Loader2 } from 'lucide-react';

export function ProfileLoadingScreen() {
  return (
    <div className="flex items-center justify-center">
      <Loader2 className="h-10 w-10 animate-spin" />
    </div>
  );
}
