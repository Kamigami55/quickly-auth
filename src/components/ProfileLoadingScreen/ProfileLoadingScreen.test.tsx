import '@testing-library/jest-dom';

import { render } from '@testing-library/react';

import { ProfileLoadingScreen } from './ProfileLoadingScreen';

describe('ProfileLoadingScreen', () => {
  it('renders loading screen unchanged', () => {
    const { container } = render(<ProfileLoadingScreen />);
    expect(container).toMatchSnapshot();
  });
});
