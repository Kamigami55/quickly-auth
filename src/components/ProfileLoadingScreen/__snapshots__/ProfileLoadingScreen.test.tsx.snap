// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileLoadingScreen renders loading screen unchanged 1`] = `
<div>
  <div
    class="flex items-center justify-center"
  >
    <svg
      class="lucide lucide-loader-circle h-10 w-10 animate-spin"
      fill="none"
      height="24"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M21 12a9 9 0 1 1-6.219-8.56"
      />
    </svg>
  </div>
</div>
`;
